hmt_escrow/job.py

```diff
@@ -1830,7 +1905,15 @@ def test__raffle_txn_retry(self):
             self.assertFalse(success)
             self.assertEqual(
                 handler_mock.call_args_list,
-                [call(txn_mock, gas_payer="1", gas_payer_priv="11", gas=6700000)],
+                [
+                    call(
+                        txn_mock,
+                        gas_payer="1",
+                        gas_payer_priv="11",
+                        gas=6700000,
+                        hmt_server_addr=None,
+                    )
+                ],
```

Comment:
New functionality has been added: e.g. starting new Jobs on two different networks. It might make sense to add dedicated tests for this. It will likely require changing get_w3 function in eth_bridge.py .